### PR TITLE
fix #73405 l.shortzzy.in

### LINKED ISSUE #73405: l.shortzzy.in
### Issue URL (Ads)
 [https://l.shortzzy.in/OaTsT7](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fl.shortzzy.in%2FOaTsT7)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#73405

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+shortzzy.*,l.shortzzy.in##.btnlink
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:257]: On my end domain is l.top20hub.com

File: EnglishFilter/sections/adservers.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||modificationdispatch.com^third-party
```
Comment (Alex-302) [inline EnglishFilter/sections/adservers.txt:8]: at the moment this is duplicate

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: What how?

File: EnglishFilter/sections/adservers.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: Okay

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:257]: image

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: Hmm let me check with other IP.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: image
Here!

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: There is anti adblock now in your domian.

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:257]: I don't see anti adblock. Added domain to previous rules to hide ad buttons.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: @Alex-302 Fake Buttons are back!
image

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: I experienced this before that's why put this rule `##.btnlink`

**Pull request closed without merging**